.travis.yml

```diff
@@ -92,6 +97,11 @@ matrix:
         - python: 3.5
           env: NUMPY_VERSION=prerelease
 
+    allow_failures:
+        # Do a PEP8 test (allow to fail unless your code completely compliant)
+        - os: linux
+          env: MAIN_CMD='pep8 packagename --count' SETUP_CMD=''
```

Comment:
Ideally yes, but last time I checked it was not yet supported, need to check whether they had a new release since.